lib/search_files.js

```diff
@@ -0,0 +1,252 @@
+var fs = require('fs');
+var path = require('path');
+var oshomedir = require('os-homedir');
+
+var isFunction = require('lodash.isfunction');
```

Comment:
Ya. IIRC the semantics around how it works aren't 100% correct back that far, but it's present all over this codebase already without issue.